scripts/build-packages-dist.sh

```diff
@@ -43,8 +43,14 @@ dirs=`echo "$targets" | sed -e 's/\/\/src\/\(.*\):npm_package/\1/'`
 # a workaround for: https://github.com/bazelbuild/rules_nodejs/issues/1219. We need to
 # do this to ensure that the version placeholders are properly populated.
 for pkg in ${dirs}; do
-  pkg_dir="${bazel_bin_path}/src/${pkg}/npm_package"
-  rm -Rf ${pkg_dir}
+  if [[ -d ${pkg} ]]; then
```

Comment:
Won't this just check if there a directory at `pwd` with the $pkg value (`material`, `cdk`, etc.)?

We need to check if the `pkg_dir`, defined in the line below, exists as a directory.  I think that as it is currently written, inside of this `if` block is essentially unreachable code, or at least unreachable in practice.

Reply:
lol, yeah I accidentally used `pkg` instead of `pkg_dir`